base/ca/src/main/java/com/netscape/ca/CRLIssuingPoint.java

```diff
@@ -3180,7 +3134,7 @@ void generateDeltaCRL(
                     e.getMessage()));
             return;
 
-        } catch (Throwable e) {
+        } catch (Exception e) {
```

Comment:
Not sure about this change (and the following similar change). With `Throwable` we are capturing any possible problem and the execution should not stop but with `Exception` some errors could be thrown stopping the execution. Generally, it is better to stop for such problematic error but in some case could be better not to stop. Is this such a case?